plugins/VisitsSummary/Controller.php

```diff
@@ -106,6 +107,17 @@ public function getEvolutionGraph(array $columns = array(), array $defaultColumn
             'avg_time_generation'
         );
 
+        $period = Common::getRequestVar('period', false);
+
+        if (SettingsPiwik::isUniqueVisitorsEnabled($period)) {
+            // add number of unique visitors if available
+            $selectableColumns = array_merge(
+                array($selectableColumns[0]),
+                array('nb_uniq_visitors'),
+                array_slice($selectableColumns, 1)
+            );
```

Comment:
If we want 'nb_uniq_visitors' to be always after 'nb_visits', maybe this is a better way (no need to modify if order of items in $selectableColumns changes):
```
$pos = array_search('nb_visits', $selectableColumns);
// array(a, b) -> array(a, x, b)
array_splice($selectableColumns, $pos+1, 0, array('nb_uniq_visitors'));
```